tests/forwarding/watch_port_test.cc

```diff
@@ -838,6 +850,12 @@ TEST_P(WatchPortTestFixture, VerifyWatchPortActionInCriticalState) {
     absl::flat_hash_set<int> expected_member_ports =
         CreateExpectedMemberPorts(members);
     expected_member_ports.erase(selected_port_id);
+
+    // Verifies the actual members inferred from receive traffic matches the
+    // expected members.
+    ASSERT_THAT(num_packets_per_port,
+                UnorderedPointwise(KeyEq(), expected_member_ports));
+
     ASSERT_OK(VerifyGroupMembersFromReceiveTraffic(num_packets_per_port,
```

Comment:
Can you remove this, merged the changed wrongly.

Reply:
Done.